packages/wrangler/src/api/types/index.ts

```diff
@@ -0,0 +1,71 @@
+import { writeFile } from "fs/promises";
+import path from "path";
+import { readConfig } from "../../config";
+import type { Config } from "../../config";
+
+export async function generateProjectTypes(options: {
+	config?: Config;
+	configFile?: string;
+	outFile?: string;
+	persistToFilesystem?: boolean;
+}): Promise<string> {
+	return generateTypes(getProjectTypes, options);
+}
+
+export async function generateRuntimeTypes(options: {
+	config?: Config;
+	configFile?: string;
+	outFile?: string;
+	persistToFilesystem?: boolean;
+}): Promise<string> {
+	return generateTypes(
+		async (config) =>
+			getRuntimeTypes({
+				compatibilityDate: config.compatibility_date,
+				compatibilityFlags: config.compatibility_flags.filter(
+					(flag) => !flag.includes("nodejs_compat")
```

Comment:
Better to be explicit here, checking against `nodejs_compat` and `nodejs_compat_v2`

Reply:
Good call.